src/Microsoft.AspNetCore.All/Microsoft.AspNetCore.All.csproj

```diff
@@ -147,4 +147,9 @@
     <PackageReference Include="Microsoft.Extensions.Logging.AzureAppServices" Version="$(AspNetCoreVersion)" PrivateAssets="None" Condition="'$(BUILD_PACKAGE_CACHE)' == 'true'" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Content Include="build\**\*.xml" PackagePath="%(Identity)" />
```

Comment:
`Package="true"` and remove `PackagePath`. Also, what's the xml file that's being added?